### PR TITLE
Adds felinid claws

### PR DESCRIPTION
**Pull request by Sparkezel**
## About The Pull Request
Adds a mutation from felinids that allows you to grow claws, they deal 16 damage, are sharp edged and make you drop items in active hand. They are better than sawblades.

port of https://github.com/tgstation/tgstation/pull/55326

## Why It's Good For The Game

Adds a fewinid mutation that awwows them to gwow cwaws to mauw peopwe

Also its a buff caused by reintroduction of felinid cocoa reaction

## Changelog
:cl:
add: Claws mutation has been found in felinid DNA
/:cl:

### CODE REVIEW

**Comment by Burgermancoding** [comment]
you need jesus

**Comment by HansRobertCoooding** [comment]
good change trust me merge now asap

**Comment by Pinkufeck** [comment]
based

**Comment by Pugie** [comment]
no

**Comment by AcapitalA** [comment]
I own a syringe gun for catgirl defense, since that's what Sparkezel intended. Four catbeasts break into my lab. "What the devil?" As I grab my inquisitor's hardsuit and cocoa-filled syinge gun. Blow four darts through the first cat-man, he's dead on the spot. Draw my reagent dart shotgun on the second cat-man, miss him entirely because it's an inaccurate PoS and nail the CMO's cat. I have to resort to the odysseus located at the mech bay loaded with chocolate, "Tally ho lads" the syringe whirlwind OD's two catbeasts in the storm, the sound and extra shrapnel set off fire alarms. Fix hypospray and charge the last terrified rapscallion. He pukes himself to death, waiting on security to arrive since chocolate OD for catpeople are impossible to heal at his stage. Just as Sparkezel intended.

File: code/datums/mutations/body.dm [hunk 1]
```diff
@@ -518,3 +518,92 @@
 
 	if(istype(new_limb, /obj/item/bodypart/head))
 		return COMPONENT_NO_ATTACH
+
+/datum/mutation/human/claws //not a spell, so doesn't use the "power" variable
+	name = "Claws"
+	desc = "An awakened mutation found in felinids that reshape the bone structure of the wrist to include claws."
```
Comment (stanalbatross) [inline code/datums/mutations/body.dm:524]:
```suggestion
	desc = "An awakened mutation found in felinids that reshapes the bone structure of the wrists to include claws."
```

**Comment by Helg2** [comment]
16 damage fists for everyone, except plasmemes. It wont end good.

**Pull request closed without merging**